core-bundle/tests/Security/Logout/LogoutSuccessHandlerTest.php

```diff
@@ -41,6 +47,8 @@ public function testRedirectsToAGivenUrl(): void
             ->willReturn(false)
         ;
 
+        $this->expectDeprecation('Since symfony/security-http 5.1: The "%s" class is deprecated, use "Symfony\Component\Security\Http\EventListener\DefaultLogoutListener" instead.');
```

Comment:
Deprecations that occur only once when the php file is autoloaded need to be handled in the *tests/PhpunitExtension/DeprecatedClasses.php* file, because the file could have already been autoloaded somewhere else and the deprecation would then not be triggered again.